src/App/Error.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\App;
+
+/**
+ * Class for constants of all possible critical errors which can stop deployment process.
+ */
+class Error
+{
+    public const BUILD_COMPOSER_PACKAGE_NOT_FOUND = 1;
```

Comment:
please, do not use code 1 - it is a common code for all errors

Reply:
Agree, updated